dashboard/config/routes.rb

```diff
@@ -149,6 +149,16 @@ module OPS
       end
     end
 
+    # /s/xxx/lockable/yyy/puzzle/zzz
+    resources :lockable_stages, only: [], path: "/lockable", format: false do
```

Comment:
afaict `param` will allow me to change the suffix, but whatever i choose will still have the `lockable_stage_` prefix. Not too sure what a better name is - maybe `lockable_stage_position` and then also change `stage_id` in the previous block to `stage_position` since in both cases these aren't really ids. The concern would be that this confuses between position and what we're currently calling url_param.

I'm also still not loving the `url_param` name in general. Any ideas for something better? `relative_position`? As you pointed out in a different comment, I can/should use this for `stage#localized_title`, however once I do so `url_param` becomes less accurate, as we're using as both a url_param and a naming mechanism.